tests/components/cloud/test_client.py

```diff
@@ -183,6 +184,44 @@ async def test_handler_google_actions_disabled(
     assert resp["payload"] == response_payload
 
 
+async def test_handler_ice_servers(
+    hass: HomeAssistant, mock_cloud_fixture: CloudPreferences
```

Comment:
This fixture is legacy. The new way is to use the `cloud` fixture and `set_cloud_prefs` fixture.

Reply:
Did I do it in a way you meant now?